tests/test-class-amp-theme-support.php

```diff
@@ -212,6 +212,54 @@ public function test_prepare_response() {
 		$this->assertInstanceOf( 'DOMAttr', $removed_nodes['onclick'] );
 	}
 
+	/**
+	 * Test validate_non_amp_theme.
+	 *
+	 * @global WP_Widget_Factory $wp_widget_factory
+	 * @global WP_Scripts $wp_scripts
+	 * @covers AMP_Theme_Support::validate_non_amp_theme()
+	 */
+	public function test_validate_non_amp_theme() {
+		global $wp_widget_factory, $wp_scripts;
+		$wp_scripts = null;
+
+		add_theme_support( 'amp' );
+		AMP_Theme_Support::init();
+		AMP_Theme_Support::finish_init();
+		$wp_widget_factory = new WP_Widget_Factory();
+		wp_widgets_init();
+
+		ob_start();
+		?>
+		<!DOCTYPE html>
+		<html lang="en-US" class="no-js">
+			<head>
+				<meta charset="UTF-8">
+				<meta name="viewport" content="width=device-width">
+				<?php wp_head(); ?>
+			</head>
+			<body>
+				<?php wp_print_scripts( 'amp-mathml' ); ?>
+			</body>
+		</html>
+		<?php
+		$original_html  = trim( ob_get_clean() );
+		$removed_nodes  = array();
+		$sanitized_html = AMP_Theme_Support::prepare_response( $original_html, array(
+			'validation_error_callback' => function( $removed ) use ( &$removed_nodes ) {
```

Comment:
I don't think we need `validation_error_callback` here since `$removed_nodes` is never checked. Alternative, you could `$this->assertEmpty( $removed_nodes )` to verify that nothing got stripped out.